### PR TITLE
myst-161 connection status connecting

### PR DESCRIPTION
**Pull request by shroomist**

### CODE REVIEW

File: client_connection/manager.go [hunk 1]
```diff
@@ -39,12 +39,12 @@ func NewManager(mysteriumClient server.Client, dialogEstablisherFactory DialogEs
 }
 
 func (manager *connectionManager) Connect(identity identity.Identity, NodeKey string) error {
-	manager.status = statusConnecting()
 	clientSession, err := manager.mysteriumClient.SessionCreate(NodeKey)
 	if err != nil {
 		manager.status = statusError(err)
 		return err
 	}
+	manager.status = statusConnecting(clientSession.Id)
```
Comment (tadovas) [inline client_connection/manager.go]: First of all sessionid comming from discovery service will be removed in the future. There is no point exposing it when status is connecting as a lot of can happen even before connection is really established. There fore status connecting WITHOUT session id was done intentionally. @Waldz your word is needed here.

File: tequilapi/endpoints/connection_test.go [hunk 2]
```diff
@@ -58,12 +58,37 @@ func TestNotConnectedStateIsReturnedWhenNoConnection(t *testing.T) {
...
+	assert.Nil(t, err)
+	resp := httptest.NewRecorder()
+
+	connEndpoint.Status(resp, req, httprouter.Params{})
+
+	assert.Equal(t, http.StatusOK, resp.Code)
+	assert.JSONEq(
+		t,
+		`{
+            "status" : "Connecting",
```
Comment (tadovas) [inline tequilapi/endpoints/connection_test.go]: Formatting.

File: client_connection/interface.go [hunk 3]
```diff
@@ -7,7 +7,7 @@ import (
 type State string
 
 const (
-	NotConnected = State("NotConnected")
+	NotConnected = State("Not Connected")
```
Comment (Waldz) [inline client_connection/interface.go]: Why spaces?

File: client_connection/manager.go (hunk 1 above)
Comment (Waldz) [reply client_connection/manager.go]: - Just dont use variable from endpoint `mysteriumClient.SessionCreate()`
- Status "connecting" should start in very beggining  of `Connect()`

File: tequilapi/endpoints/connection_test.go [hunk 4]
```diff
@@ -63,7 +63,30 @@ func TestNotConnectedStateIsReturnedWhenNoConnection(t *testing.T) {
...
+	req, err := http.NewRequest(http.MethodGet, "/connection", nil)
+	assert.Nil(t, err)
+	resp := httptest.NewRecorder()
+
+	connEndpoint.Status(resp, req, httprouter.Params{})
+
+	assert.Equal(t, http.StatusOK, resp.Code)
+	assert.JSONEq(
+		t,
+		`{
```
Comment (Waldz) [inline tequilapi/endpoints/connection_test.go]: Formating

File: tequilapi/endpoints/connection_test.go [hunk 5]
```diff
@@ -63,7 +63,30 @@ func TestNotConnectedStateIsReturnedWhenNoConnection(t *testing.T) {
 		resp.Body.String())
 }
 
-func TestConnectedStateAndSessionIdIsReturnedWhenIsConnection(t *testing.T) {
+func TestConnectedStateAndSessionIdIsReturnedWhenIsConnecting(t *testing.T) {
```
Comment (Waldz) [inline tequilapi/endpoints/connection_test.go]: Wrong name of test

**Pull request merged**